c++/src/capnp/serialize-text.h

```diff
@@ -22,9 +22,9 @@
 #pragma once
 
 #include <kj/string.h>
-#include "dynamic.h"
-#include "orphan.h"
-#include "schema.h"
+#include "capnp/dynamic.h"
```

Comment:
Please don't use quoted syntax for absolute-path includes. These should use `<>` syntax. I know that Google feels otherwise but they are crazy. We have used `<>` for all such includes consistently in KJ and Workers code.

Reply:
Updated the includes